### PR TITLE
EWL-3485 Hub | Visual QA Fixes

### PR DESCRIPTION
**Pull request by chelsiejohnston**
## JIRA Ticket(s)
- [EWL-3485: Hub | Visual QA Fixes](https://issues.ama-assn.org/browse/EWL-3485)

## Description:
Various CSS changes in accordance with visual QA.

These changes mirror changes on the Corporate site: https://github.com/AmericanMedicalAssociation/AMA-Corporate-site/pull/1032

## To Test:
Navigate to Templates > Hub > Hub and verify changes are correct according to visual QA:
- [x] Primary CTA top/bottom padding should be same as `.button-small`.
- [x] `.hub-hero-cta-only` padding top/bottom 1.5em -> 1em.
- [x] `.hub-hero-button-container` margin top/bottom 20px -> 10px.
- [x] On Image & Text cards `.hub-card-image + .hub-card-title` should get `margin-top:auto` so that all card CTAs float to bottom.
- [x] `.hub-header-small` (secondary text and CTA) get right aligned, remove min-width
- [x] Ribbon should be sticky to the top of page (desktop)

## Automated Test:
N/A (why does the SG template have a section for this?)

## Relevant Screenshots/GIFs:
3485-sg-1
3485-sg-2

## Remaining Tasks:
- Investigate heights of row cards in IE11 - broken out into [EWL-3518](https://issues.ama-assn.org/browse/EWL-3518)

## Additional Notes:
N/A

---

[Pull Request Guidelines](https://github.com/palantirnet/development_documentation/blob/master/guidelines/pr_review.md)

### CODE REVIEW

**Comment by rpama** [comment]
Overall I see no issues with merging this, but lacking background on this task, I would appreciate feedback first on two small questions. One is already noted as a comment about the 1000px breakpoint. The other is a gif I'm attaching here about the expected behavior of the `.hub-header_small` in non-desktop sizes. It's worth noting these are both questions about pre-existing issues, but figured they're worth trying to answer here and now if we have time.
hub-header_small

**Comment by froboy** [comment]
@rpama the phone number behavior as you documented is as requested... As for the breakpoint behavior I'm not sure.

File: styleguide/source/_patterns/02-organisms/03-sections/01-card-row/_card-row.scss [hunk 1]
```diff
@@ -1,5 +1,5 @@
 .hub-card-row,
 .card-row {
-  margin-top: 24px;
+  //margin-top: 24px;
```
Comment (froboy) [inline styleguide/source/_patterns/02-organisms/03-sections/01-card-row/_card-row.scss]: @chelsiejohnston please remove this line if it's not needed.

File: styleguide/source/_patterns/02-organisms/04-hub/00-hub-header/_hub-header.scss [hunk 2]
```diff
@@ -7,6 +7,18 @@
 .hub-header {
   padding: 0 25px;
   background-color: $purple;
+
+    @include breakpoint(1000px) {
```
Comment (froboy) [inline styleguide/source/_patterns/02-organisms/04-hub/00-hub-header/_hub-header.scss]: let's go with `$bp-med: 56.25em; // 900px` as discussed in Slack.

**Comment by froboy** [comment]
@chelsiejohnston apart from those comments above, this looks awesome. The code changes are nice and small and elegant.

**Comment by froboy** [comment]
LGTM!

**Pull request merged**